### PR TITLE
Adds Vulpkanin Species Flag

### PR DESCRIPTION
**Pull request by KasparoVy**
No idea why I did this so late in the night but w/e. Felt Vulpkanin deserved their own flag after being in for so long and with so many people playing as them, so I tried to do a good job.

I figured a fox-type guy would be a good bet so that's what I went with for the figurehead. If you want a real close-up look at the sprites, download the DMI and zoom right in.

Adds the Vulpkanin species flag to the cargo vendor and fixes a couple typos in the item descriptions. I checked the lore to make sure those were correct.

**Rolled flag:**
vulpflag3

**Unrolled flag:**
vulpflag2

**Testing:**
vulpflag1

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
:+1:  x :100: 

That's awesome and cute--I like.

File: code/game/objects/items/flag.dm [hunk 1]
```diff
@@ -148,7 +148,7 @@
 
 /obj/item/flag/species/taj
 	name = "Tajaran flag"
-	desc = "A flag proudly proclaiming the superior heritage of Tajaran."
+	desc = "A flag proudly proclaiming the superior heritage of Tajara."
```
Comment (Fox-McCloud) [inline code/game/objects/items/flag.dm]: This is mostly a Bay thing--here, there's generally just called "Tajaran".

**Comment by KasparoVy** [comment]
Sorry for the ridiculously late response there, I put up the PR and went straight to bed haha.

**Comment by Fox-McCloud** [comment]
@KasparoVy 

You may want to make in-hands for this. It's by no means necessary, but I believe all the others have in-hands.

**Comment by KasparoVy** [comment]
The ones I used as direct references for some of the sprite's main features-- i.e. Tajaran and Vox did not have in-hand sprites. Would it be alright if I made a separate PR that sorted out all the in-hand sprites for all the flags instead of lumping it in and dramatically widening the scope?

**Comment by Fox-McCloud** [comment]
@KasparoVy 

That's fine; don't feel obliged that you have to, and yes, that would probably be best for a separate PR.

**Comment by KasparoVy** [comment]
Sweet, alright I guess that's all I've got for this PR.

**Comment by Iamgoofball** [comment]
:-1:

**Pull request merged**